core/Api/Handlers/QueryController.cs

```diff
@@ -37,6 +37,7 @@ public QueryController(QueryHandler queryHandler)
 
         #region Public Methods and Operators
 
+        [HttpGet]
```

Comment:
Note, these need to be explicit for the latest Swagger to discover them otherwise swagger fails to produce the swagger.json file and complains about an ambiguous http verb.